slatwall.php

```diff
@@ -33,7 +33,7 @@
  */
 define('SLATWALL_VERSION', '1.0.0' );
 define('SLATWALL_PLUGIN_DIR', plugin_dir_path( __FILE__ ) );
-define('AUTHORIZATION','');
+define('AUTHORIZATION','dGVuMjQ6NDBCbHVlOTY=');
```

Comment:
```suggestion
define('AUTHORIZATION','');
```

Reply:
Please also revert this, this project is public and this basic auth doesn't apply to anyone else